### ISSUE TITLE
QDate slide transitions broken

### ISSUE THREAD

**Issue by duncan-oxd** [quasarframework/quasar#17183]
### What happened?

The QDate component in the current docs doesn't have the expected slide transition. Instead the numbers flicker before changing.

### What did you expect to happen?

When you switch months or years, the "calendar" part of the component should do a slide animation to the left or right.

### Reproduction URL

https://codepen.io/oxd_duncan/pen/BaeaXjm

(and also https://quasar.dev/vue-components/date#basic )

### How to reproduce?

Visit the Quasar docs for version 2.16.2, in the example components, click the left or right arrow buttons next to the months or years.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome

### Quasar info output

```Shell
N/A, it's happening on the website
```

### Relevant log output

```Shell
N/A, it's happening on the website
```

### Additional context

It worked fine in 2.15, but I refreshed the docs to 2.16.2 today and now the slide transition is broken.

**Comment by duncan-oxd** [comment]
Updated with a codepen reproduction to avoid auto-closing 😇

**Comment by Lbyn96** [comment]
I've noticed that issue too.😫

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.16.3
